packages/kernel/src/kernel.ts

```diff
@@ -256,6 +254,23 @@ export abstract class BaseKernel implements IKernel {
     this._sendMessage(message);
   }
 
+  /**
+   * Send a `input_request` message to the client.
+   *
+   * @param content The input_request content.
+   */
+  protected _inputRequest(content: KernelMessage.IInputRequestMsg['content']): void {
```

Comment:
No `_` prefix if it's `protected`, otherwise looking good!

Reply:
Doing!